### DISCUSSION TITLE
fooocus 2.1.37  report an error:safetensors_rust.SafetensorError: Error while deserializing header: MetadataIncompleteBuffer

### DISCUSSION THREAD

**Discussion by xtcm121** [lllyasviel/Fooocus#626] [Q&A]
report an error:safetensors_rust.SafetensorError: Error while deserializing header: MetadataIncompleteBuffer

**Comment by Phrixus2023** [comment]
redownload model

**Reply by xtcm121** [reply]
How do I download the model again? Do you need to delete the current model? What is the path?

**Comment by bmadgett** [comment]
Also having this issue when trying the new Image Prompt feature. Have let it download the controlnet models itself (ip-adapter-plus_sdxl_vit-h.bin and fooocus_ip_negative.safetensors) and get the error. Also downloaded those separately, multiple times, every time I get the same MetadataIncompleteBuffer exception when trying Image Prompt. I can't see it's bad downloads every single time (and the other much larger standard checkpoints models are working fine).

**Reply by bmadgett** [reply]
Ok - have got it working. Didn't realise it also wanted models/clip_vision/clip_vision_vit_h.safetensors model and that one needed redownloading as well. Seems to work now.

**Reply by xtcm121** [reply]
How do I download the model again? Do you need to delete the current model? What is the path?
